src/python/deepgnn/graph_engine/snark/decoders.py

```diff
@@ -275,7 +283,7 @@ def decode(self, line: str) -> Iterator[Tuple[int, int, int, float, list]]:
 
             features.append(self._get_feature(key, length, data))
 
-        yield src, dst, typ, weight, features
+        yield src, dst, typ, weight, features, None, None
```

Comment:
Edge list should support temporal graph not just json.

Reply:
Added support to edge list